flink-end-to-end-tests/flink-tpcds-test/src/main/java/org/apache/flink/table/tpcds/utils/TpcdsResultComparator.java

```
@@ -0,0 +1,232 @@
+					}
+					expectedTop8Line.add(expectedLine);
+					expectedTop8Line.add(expectedLine2);
+
+					actualTop8Line.add(actualLine);
+					while (actualLineNum < 8) {
+						actualTop8Line.add(actualReader.readLine());
+						actualLineNum++;
+					}
+					for (int i = 0; i < 8; i++){
+						if (!isEqualLine(expectedTop8Line.get(i), actualTop8Line.get(i), i)) {
+							return false;
+						}
+					}
+					continue;
+				}
+				if ("77.ans".equals(expectedFile.getName()) && expectedLineNum == 2) {
+					String expectedLine3 = expectedReader.readLine();
+					String actualLine3 = actualReader.readLine();
+					expectedLineNum++;
+					actualLineNum++;
+
+					if (isEqualLine(expectedLine, actualLine, 2) && isEqualLine(expectedLine3, actualLine3, 3)) {
+						continue;
+					}
+					if (isEqualLine(expectedLine, actualLine3, 2) && isEqualLine(expectedLine3, actualLine, 3)) {
+						continue;
+					}
+					return false;
+				}
+
+				if ("79.ans".equals(expectedFile.getName()) && expectedLineNum == 18) {
+					String expectedLine19 = expectedReader.readLine();
+					String actualLine19 = actualReader.readLine();
+					expectedLineNum++;
+					actualLineNum++;
+
+					if (isEqualLine(expectedLine, actualLine, 18) && isEqualLine(expectedLine19, actualLine19, 19)) {
+						continue;
+					}
+					if (isEqualLine(expectedLine, actualLine19, 18) && isEqualLine(expectedLine19, actualLine, 19)) {
+						continue;
+					}
+					return false;
+				}
+
+				if (!isEqualLine(expectedLine, actualLine, expectedLineNum)) {
+					return false;
+				}
+			}
+
+			while (expectedReader.readLine() != null) {
+				expectedLineNum++;
+			}
+			while (actualReader.readLine() != null) {
+				actualLineNum++;
+			}
+
+			if (expectedLineNum != actualLineNum) {
+				System.out.println(
+					"[ERROR]incorrect number of lines! Expecting " + expectedLineNum +
+						" lines, but found " + actualLineNum + " lines.");
+				return false;
+			}
+		} catch (Exception e) {
+			e.printStackTrace();
```

Is it possible to let the exceptions bubble up and simply let the process exit with the stack trace?